src/styles/utils/mixins.scss

```diff
@@ -0,0 +1,6 @@
+@mixin text-secondary() {
+  font-size: 10px;
+  line-height: 14px;
+  font-weight: 400;
+  color: $secondary-text-color;
```

Comment:
Ensure that the variable `$secondary-text-color` is defined in `variables.scss` as per the checklist. If it's not defined, it could lead to a compilation error.